src/sentry/models/releases/release_project.py

```diff
@@ -33,8 +33,28 @@ def _on_post(project, trigger):
         ):
             schedule_invalidate_project_config(project_id=project.id, trigger=trigger)
 
+    def _subscribe_project_to_alert_rule(project, release, trigger):
+        """
+        TODO: potentially enable custom query_extra to be passed on ReleaseProject creation (on release/deploy)
+        """
+        from django.utils import timezone
+
+        from sentry.incidents.models import AlertRule
+        from sentry.incidents.utils.types import AlertRuleActivationConditionType
+
+        query_extra = f"release:{release.version} AND event.timestamp:>{timezone.now().isoformat()}"
```

Comment:
@wedamija

Can you validate whether this query makes sense?

also cc @lynnagara or @untitaker ?

Reply:
Yes, that query looks right. SNS folks don't have any context around our text query language, but this will convert into the appropriate snql.

Just make sure there's a test that validates that this correctly manages to create a subscription using this in snuba. Might also be worth using `SENTRY_SNUBA_INFO=1` to print out the query and make sure it make sense.